draft-ietf-ivy-network-inventory-yang.md

```diff
@@ -164,55 +164,58 @@ The YANG data model defined in this document conforms to the Network Management
 
 ## Terminology
 
-  The following terms are defined in {{!RFC7950}} and are not
-  redefined here:
+The following terms are defined in {{!RFC7950}} and are not redefined here:
+- client
+- server
+- augment
+- data model
+- data node
 
-  *  client
+The following terms are defined in {{!RFC6241}} and are not redefined here:
+- configuration data
+- state data
 
-  *  server
+The following terms are defined in {{IANA_YANG}} and are not redefined here:
+- backplane
+- battery
+- container
+- cpu
+- chassis
+- fan
+- module
+- port
+- power supply
+- sensor
+- stack
+- storage device
 
-  *  augment
+> Note that the definition of port component in {{IANA_YANG}} needs to be refined in future version of this document.
 
-  *  data model
+> TBD: Recap the concept of chassis/slot/component/board/... in {{TMF_SD2-20}}.
 
-  *  data node
+Also, the document makes use of the following terms:
 
-  The following terms are defined in {{!RFC6241}} and are not redefined
-  here:
+Network Inventory:
+: A collection of data for network elements and their components managed by a specific management system.
 
-  *  configuration data
+Physical Network Element:
+: An implementation or application specific groupings of components (e.g., hardware components).
 
-  *  state data
+Network Element:
+: The generalization of the physical network element definition to include other types of inventory objects which can be managed, from an inventory perspective, like physical network elements.
```

Comment:
If you "include other types of inventory objects" you probably point to "other components".
Why do not say "The generalization of the physical network element definition to include other types of inventory objects which can be managed, from an inventory perspective, like components of physical network elements."

Reply:
```suggestion
: The generalization of the physical network element definition.
```